modules/tests/shared/src/test/scala-3/io/circe/ConfiguredDerivesSuite.scala

```diff
@@ -420,11 +420,6 @@ class ConfiguredDerivesSuite extends CirceMunitSuite:
       List(DownField("ConfigExampleFoo"))
     )
     assert(Decoder[ConfigExampleBase].decodeJson(json) === Left(failure("unexpected fields: anotherField")))
-    assert(
-      Decoder[ConfigExampleBase].decodeAccumulating(json.hcursor) === Validated.invalidNel(
-        failure("unexpected field: anotherField")
-      )
-    )
```

Comment:
Why did you remove this test?

Reply:
I apologize for the oversight. I mistakenly removed the above expression, under the false impression that `ConfiguredDecoder[ConfigExampleBase]#decodeAccumulating` would be called.